script.colorbox/resources/lib/Utils.py

```diff
@@ -889,6 +911,46 @@ def log(txt):
     if isinstance(txt, str):
         txt = txt.decode("utf-8")
     message = u'%s: %s' % (ADDON_ID, txt)
-    xbmc.log(msg=message.encode("utf-8"), level=xbmc.LOGDEBUG)
+    xbmc.log(msg=message.encode("utf-8"), level=xbmc.LOGNOTICE)
```

Comment:
Not sure this change is advisable... It can flood a users logs with unneeded entries. Any reason you changed it from LOGDEBUG?

Reply:
Yeah I can change that, it was just during debugging for users. Will revert thanks :)